sgt-back-end/server.js

```diff
@@ -0,0 +1,171 @@
+import express from 'express';
+import pg from 'pg';
+
+const app = express();
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+      select *
+        from "grades"
+    `;
+    const result = await db.query(sql);
+    res.status(200).json(result);
+  } catch (err) {
+    console.error(err);
+    res.status(500).json({ error: 'An unexpected error occurred.' });
+  }
+});
+
+app.post('/api/grades', async (req, res) => {
+  try {
+
+    let error = {};
```

Comment:
Everywhere you assign to `error` you use it immediately. It would be cleaner to remove this declaration and just declare it each time you assign to it.